### ISSUE TITLE
Minor PHP version and missing php.ini error

### ISSUE THREAD

**Issue by rubenvarela** [lando/acquia#87]
Test `.lando.yml` file.

```yml
name: 20240308-lando-lamp
recipe: acquia
config:
  ah_application_uuid: 123
  ah_site_group: 123
  webroot: .
  xdebug: 'develop,debug'
```

To reproduce, `lando start`,

Error:

```
Box Requirements Checker
========================

> Using PHP 7.4.33
> PHP is using the following php.ini file:
  WARNING: No configuration file (php.ini) used by PHP!

> Checking Box requirements:
  E................



 [ERROR] Your system is not ready to run the application.




Fix the following mandatory requirements:
=========================================

 * The application requires a version matching "^8.1".
```

This is probably coming from acli, https://github.com/acquia/cli/blob/a52dddc14cc615b32f4e8daf113918fd217eb3af/composer.json#L15

If we are defaulting to latest, https://github.com/lando/acquia/blob/25b8ee176a51bb54fe1475584085ba5fb812bc68/docs/config.md#L15, we should probably update the minimum version from 7.4.

Personally, I think tracking the newest version at Acquia and allowing downgrading with user interaction would be a good option. This would allow developing and testing against the 'latest and greatest' and find issues before they arise on prod.

**Comment by reynoldsalec** [comment]
@rubenvarela agreed; noticed this when I was working on some of the config as well.

Looks like Acquia currently uses 8.1 as the default, you want to take a stab at updating that @rubenvarela? I think you just need to update the default value in the `builders/acquia.js` and then make sure any of the tests in the `examples` directory that check PHP version and don't have it overriden in the `.lando.yml` are looking for 8.1, not 7.4. I don't mind doing it, so no worries if you're busy, just let me know!

**Comment by rubenvarela** [comment]
@reynoldsalec for sure. Happy to send the PR for this.

**Comment by rubenvarela** [comment]
@reynoldsalec when you get a sec. I have a PR for this for you to review.

**Comment by rubenvarela** [comment]
Closed by #94
